README.md

```diff
@@ -41,16 +41,9 @@ OpenStack Liberty Summit, we will be moving this to
 
 0. You must have a [devstack setup running on a single VM](http://docs.openstack.org/developer/devstack/guides/single-vm.html).
 1. [Download and source the project specific rc file](http://docs.openstack.org/user-guide/common/cli_set_environment_variables_using_openstack_rc.html).
-2. `cd don; python collector.py` - this generates don.json. Update the `myenv` dictionary in
-   don/collector.py to match the settings in the file downloaded in the previous
-   step. **This step is soon going to be automated and integrated with the next
-   step.**
-3. `python manage.py runserver 0.0.0.0:8000.` This runs the django server that
-   accepts requests on port 8000 from any IP address. For a better understanding of
-   how to start the django server, please look at official django documentation.
-   If you get an error "Error: That port is already in use." and you are sure
-   that there is nothing listening on the port, then use `sudo fuser -k 8000/tcp`
-   to kill any process that might still be listening on that port.
+2. Copy the DON source to Horizon directory.
+3. `python manage.py collectstatic.` This collects the needed static files for Horizon dashboard.
```

Comment:
Do we still have this "additional" step? I thought you had integrated everything to be from inside the Horizon dashboard.